### ISSUE TITLE
Bookmarks + Widget Malfunctioning

### ISSUE THREAD

**Issue by PassionateG1t** [bromite/bromite#2214]
### Preliminary checklist

- [X] I have read the [README](https://github.com/bromite/bromite/blob/master/README.md).
- [X] I have searched the existing issues for my problem. This is a new ticket, NOT a duplicate or related to another open issue.
- [X] I have read the [FAQs](https://github.com/bromite/bromite/blob/master/FAQ.md).
- [X] I have updated Bromite to the latest version. The bug is reproducible on this latest version.
- [X] This is a bug report about the Bromite browser; not the website nor F-Droid nor anything else.

### Can the bug be reproduced with corresponding Chromium version?

Yes

### Bromite version

103.0.5060.140

### Device architecture

arm64-v8a

### Android version

12.0

### Device model

SM-A515F/DSN

### Changed flags

#enable-parallel-downloading
#enable-tab-groups
#strict-origin-isolation
#context-menu-google-lens-chip
#context-menu-search-with-google-lens
#context-menu-shop-with-google-lens
#context-menu-translate-with-google-lens
#google-lens-sdk-intent
#enable-prerender2
#omnibox-trigger-for-prerender2
#enable-throttle-display-none-and-visibility-hidden-cross-origin-iframes

### Is this bug about the SystemWebView?

No

### Is this bug happening in an incognito tab?

No

### Is this bug caused by the adblocker?

No

### Is this bug a crash?

No

### Describe the bug

The bookmarks widget stops properly interacting; I can't exit/open a folder but can only click on the bookmarks that are present in the last used folder.

As for the bookmarks section when entering the app, it loads infinitely, so I cannot open any bookmark. 

Basically, I can't navigate my bookmarks.

Temporary fix: force stopping the app.

### Steps to reproduce the bug

1. Choose any bookmark via the widget
2. Exit website by swiping back
3. Remove Bromite from recent apps

### Expected behavior

Widget (and bookmarks section) functioning normally after ending Bromite session.

**Comment by kokukyocho** [comment]
Hello @PassionateG1t!

I am a bot :robot:

Thanks for submitting this issue!
Since this bug is reproducible with Chromium please report it upstream on the [Chromium issue tracker](https://bugs.chromium.org/p/chromium/issues/list) and do not forget to read [Chromium project bug reporting guidelines](https://www.chromium.org/for-testers/bug-reporting-guidelines) first.
